### DISCUSSION TITLE
更新日志 - 20230414

### DISCUSSION THREAD

**Discussion by jjm2473** [istoreos/istoreos#713] [Announcements]
# 设备：
* rk3568机型：内核开启 ipv4 的策略路由 #584 #703
* r4s/r2s/rpi4：从固件移除部分用不上的内核模块（可能只有x86平台才能用），如有需要可从软件包安装或者提issue
* r2s：修复 LAN 口相关的多个问题：部分机器不识别，频繁掉线，跑不满千兆。 #690 #696

**Comment by jjm2473** [comment]
x86 和 rk3568 机型 有 22.03.4 的测试固件，详情看 https://github.com/istoreos/istoreos/issues/716 。
x86 测试固件支持了 N5105 的硬解，从iStore安装jellyfin插件，并且部署以后，在jellyfin的“控制台”-“播放”选择 QuickSync 转码器，剩下的硬解选项除了 AV1 以外都可以勾选上。

注意事项：
1. ~测试固件尚未支持 `FullconeNAT` ，不要勾选，也不要提issue，之后会支持。~ (已经支持)
2. 不要使用在线升级，不然会降级到线上正式版本。
3. 如果之前使用其他版本固件，刷机前备份，尽量不保留配置升级。
4. 要反馈问题到 https://github.com/istoreos/istoreos/issues/716 跟帖，不要单独开issue

**Reply by toplezs** [reply]
i'm using x86_64_efi, after build 20230320 client can't get IP router

**Reply by Eircodong** [reply]
请问x86的22.03.4的测试固件在哪里下载

**Reply by jjm2473** [reply]
还在等发布，一般测试固件在 https://fw.koolcenter.com/iStoreOS/alpha/ 下载

**Reply by jjm2473** [reply]
上传完了

**Reply by jjm2473** [reply]
> i'm using x86_64_efi, after build 20230320 client can't get IP router

Maybe wrong port, and DHCP server conflict.

1. disconnect all network cables
2. reboot
3. PC connect to a network port, if not IP, try another port

**Reply by zric** [reply]
R2S在线更新就挂了 要重刷固件

**Reply by jjm2473** [reply]
> R2S在线更新就挂了 要重刷固件

应该跟这次更新无关，都没改固件的结构和分区，只是升级驱动和配置

**Comment by FSDXC111** [comment]
R2C依旧无法突破500M

**Reply by jjm2473** [reply]
理论上应该跟r2s一样，不过我也没r2c测试。

**Reply by FSDXC111** [reply]
反正之前用骷髅头是可以跑到八九百，就换到Istore之后咋都上不去

**Comment by tinobara** [comment]
更新后无线网卡挂了,移除后新增提示报错
TypeError
Cannot read properties of undefined (reading 'value')
![VKE$LLTG9FU}}Z8SZAI0G44](https://user-images.githubusercontent.com/57656654/233032502-cb9f9cd0-9ff6-460e-8b2a-c1f0cfcd706b.png)

**Reply by tinobara** [reply]
机器是R4S,网卡是华硕AC1300

**Reply by tinobara** [reply]
重新卡刷新版本没有了无线选项,刷回老版本0320的有无线选项并且可以正常使用

**Reply by jjm2473** [reply]
具体型号是什么，还有执行 `lsusb -tv` 看看。应该是清理内核模块删掉了

**Reply by jjm2473** [reply]
不知道你这个网卡是什么芯片。如果还需要支持这个网卡，提个issue，issue里面包括`lsusb -tv`的结果，最好在正常使用网卡的固件里执行`lsusb -tv`。
